netbox/templates/home.html

```diff
@@ -1,6 +1,18 @@
 {% extends '_base.html' %}
 {% load helpers %}
 
+{% block header %}
+    {% if new_release %}
```

Comment:
I think we need to include `{{ block.super }}` here to ensure we don't overwrite any existing content (e.g. an error message from a previous request).